### PR TITLE
Murphya boat

### PR DESCRIPTION
**Pull request by murphyaye**
I think my code is ready whenever you want to look over it.

### CODE REVIEW

File: hw03_murphya.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Author: Aaliyah Murphy
+# Username: Murphya
+#
+# Assignment: Hw03
+# Google Doc Link:https://docs.google.com/document/d/1QOq5mSUdxNJacGKRdWqrQEk5mgNh5cuqwZO52XxC4JY/edit?usp=sharing
+#
```
Comment (sheggen) [inline hw03_murphya.py:6]: Use the header block format from the template.

File: hw03_murphya.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+abby.left(90)
+abby.end_fill()
+
+abby.penup()
+abby.goto(-40, -65)
+abby.pendown()
+#This code makes the ship dock
+abby.forward(250)
+abby.left(90)
+abby.color("#8E5B3A", "#8E5B3A")
```
Comment (sheggen) [inline hw03_murphya.py:43]: All of this should be at the beginning of `main()`. There should be no code at the highest indentation level. 

If you are having scoping issues, then you probably need to pass a turtle into functions:

```
def draw_something(turt):
    turt.forward(10)
    ...

def main():
    t = turtle.Turtle()
    draw_something(t)
```

File: hw03_murphya.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+def drawRectangle(t, w, h):
```
Comment (sheggen) [inline hw03_murphya.py:44]: Missing the docstring

File: hw03_murphya.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+aaliyah.begin_fill()
+aaliyah.right(90)
+aaliyah.circle(100,180)
+aaliyah.left(90)
+aaliyah.forward(200)
+aaliyah.end_fill()
+aaliyah.backward(100)
+aaliyah.right(90)
+aaliyah.forward(75)
+aaliyah.backward(15)
```
Comment (sheggen) [inline hw03_murphya.py:67]: This would probably make a good function.

**Comment by murphyaye** [comment]
I fixed all of your comments and am good for review.

**Pull request merged**